index.d.ts

```diff
@@ -0,0 +1,104 @@
+// TypeScript Version: 3.7
+
+import {Test} from 'unist-util-is'
+import * as unist from 'unist'
+
+export {}
+
+type Null = undefined | null
+
+declare namespace messageControl {
+  export {}
+  /** a comment marker */
+  export interface Marker<N extends unist.Node> {
```

Comment:
`export`  in namespace is useless. Please delete it.

Reply:
same as above